src/frontend/src/tests/lib/canisters/backend.canister.spec.ts

```diff
@@ -27,6 +28,14 @@ vi.mock(import('$lib/constants/app.constants'), async (importOriginal) => {
 	};
 });
 
+vi.mock(import('$lib/actors/agents.ic'), async (importOriginal) => {
```

Comment:
The tests were green because the backend/signer calls are correctly mocked, but since we are using the "real" Canister.create methods [which call getAgent here](https://github.com/dfinity/oisy-wallet/blob/main/src/frontend/src/lib/canisters/signer.canister.ts#L21), we were getting those warnings.

Reply:
Gotcha!

Reply:
Thanks for the explanation.